.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,27 @@
+name: CI
```

Comment:
can we rename both actions so it's easy to tell them apart?

Reply:
consolidated unit test to workflow.yaml